docs/community/roadmap.md

```diff
@@ -151,10 +151,29 @@ Investigate to cooperate with argo to support job dependencies.
 * Issue: https://github.com/volcano-sh/volcano/issues/1627
 * Owner: @hwdef @shinytang6 @Thor-wl
 
-### Support configuration hot update
+### Support configuration hot update(Delay)
 * Description: Add hot update for Volcano components arguments.
 * Issue: https://github.com/volcano-sh/volcano/issues/1326
 
+## v1.6
+### Support Dynamic Scheduling Based on Realtime Load
+* Description: Support dynamic scheduling based on realtime load.
+* Priority: High
+* Issue: https://github.com/volcano-sh/volcano/issues/1777
+* Owner: @william-wang
```

Comment:
let's add a shepherd for each features, the shepherd should be one of approvor :)

Reply:
> let's add a shepherd for each features, the shepherd should be one of approvor :)

OK. Let me add one for the last feature.

Reply:
Yep, this is a good idea.